projects/1.20/assets/enchantable-boat/enchantableboat/lang/zh_cn.json

```
@@ -0,0 +1,14 @@
+{
+  "container.enchanted_chest_boat": "运输船",
+  "enchantment.enchantableboat.climbing": "攀爬",
+  "enchantment.enchantableboat.curse_of_decay": "朽坏诅咒",
+  "enchantment.enchantableboat.floating_on_lava": "熔岩飘浮",
+  "enchantment.enchantableboat.gliding": "滑翔",
+  "enchantment.enchantableboat.ground_adaptation": "地面适应",
+  "enchantment.enchantableboat.high_capacity": "高容量",
+  "enchantment.enchantableboat.magick_boost": "魔法提速",
+  "enchantment.enchantableboat.storm_rider": "风暴骑手",
+  "enchantment.enchantableboat.swift_row": "迅捷划船",
```

```suggestion
  "enchantment.enchantableboat.swift_row": "速划",
```

```suggestion
  "enchantment.enchantableboat.swift_row": "高速推动",
```

原文`高速推進`，原翻有点生硬，考虑换词或日文直译